src/main/java/com/kodilla/ecommercee/UserMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.ecommercee;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+    public User mapToUser(final UserDto userDto) {
+        return new User(
```

Comment:
W kontrolerze przyjmujesz UserDto... a tutaj mapujesz na User, czy tak na pewno ma być? Jeśli tak to uzasadnij.